### ISSUE TITLE
[BUG] admin panel not functioning properly

### ISSUE THREAD

**Issue by leggewie** [BoltAuth/Auth#18]
I think I've reported this as bug 70 when BoltAuth was still bolt/Members.  It seems to have survived the transition to BoltAuth.

I have a pretty virgin installation of bolt 3.3.6 with BoltAuth 3.0.1 at http://localhost/bolt.  When I visit http://localhost/bolt/bolt/extensions/auth the sidebar on the right is mostly not functioning.  None of the buttons of "Enable", "Disable", "Add role" and "Remove role" seem to have any effect. There is no page refresh or any other discernible action even when I select a user first.

I also find this isn't the best UI.  It is my understanding that for those buttons to have any effect, one needs to select at least one user from the list first.  As such, I think it would help if they were greyed out until that is the case.  I hope this would be easy enough to implement.

FWIW, I use Firefox 56 on Ubuntu.  I've tried this in Chromium for the same result.

**Comment by jimi75** [comment]
Same Issue for me. 

Bolt: 3.3.6
BoltAuth: 3.0.1

Is a JS ejecution problem. I get this in console:
![screen shot 2017-10-19 at 11 26 53](https://user-images.githubusercontent.com/1430380/31764153-a937674a-b4c0-11e7-8966-9948694bd813.png)

The 'auth-admin.js' js file is laoded correctly but "auth" is not defined. Namespace problem?

**Comment by fabani** [comment]
I have the same problem with BoltAuth: 3.0.1,  its the javascript variable `baseurl` that is wrong. 
In `templates/admin/auth.twig` on row 120:
```
    <script>
        var baseurl = "{{ paths.bolt }}extend/auth/action";
```

However this seems to be fixed i the master branch.

**Comment by SvanteRichter** [comment]
Hey, sorry it's taken me a while to tag a release with this fix, it should work in 3.0.2 which I just tagged.
